src/MessageReader.test.ts

```diff
@@ -426,9 +426,9 @@ IDL: geometry_msgs/msg/Vector3
 module geometry_msgs {
   module msg {
     struct Vector3 {
-      long double x;
```

Comment:
Where did we originally get these from?

Reply:
I manually translated them from the `.msg` schemas